### PR TITLE
fix(clowder): RHICOMPL-2648 provide the secret for slack notifications

### PR DESCRIPTION
**Pull request by skateman**
## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [x] General Coding Practices

### CODE REVIEW

**Review by marleystipich2 — APPROVED** [review]
Looks good to me!

File: clowdapp.yaml [hunk 1]
```diff
@@ -69,6 +69,12 @@ objects:
         # Should be a secret ?
         - name: SECRET_KEY_BASE
           value: "secret_key_base"
+        - name: SETTINGS__SLACK_WEBHOOK
+          valueFrom:
+            secretKeyRef:
+              name: compliance-backend
```
Comment (vkrizan) [inline clowdapp.yaml:75]: Where is the secret defined?

**Review by vkrizan — APPROVED** [review]
lgtm

File: clowdapp.yaml (hunk 1 above)
Comment (skateman) [reply clowdapp.yaml:75]: Vault, it's being pulled via app-interface.

**Pull request merged**